lib/src/bazelify/build.dart

```diff
@@ -282,14 +289,33 @@ class DartWebApplication implements DartBuildRule {
         '    script_file = "$scriptFile",\n'
         '    deps = _PUB_DEPS';
     if (includeLibraries.isEmpty) {
-      return '$buffer,\n)';
+      buffer = '$buffer,\n)';
     } else {
-      return buffer +
-          ' + [\n' +
+      buffer += ' + [\n' +
           includeLibraries
               .map/*<String>*/((l) => '        ":${l.name}",\n')
               .join() +
           '    ],\n)';
     }
+    buffer += '\ndev_server(\n'
```

Comment:
Future I'd rather just make this an actual class with a string buffer, but got lazy :)